sematic/ui/packages/main/src/types/hugging_face_stored_model.tsx

```diff
@@ -0,0 +1,14 @@
+import { CommonValueViewProps } from "./common";
```

Comment:
![Screenshot 2023-06-21 at 3 11 57 PM](https://github.com/sematic-ai/sematic/assets/429433/4590364b-ed2e-4e9f-90fd-c4a848fbf3cb)
Something's up here. The chevron is misplaced and the model name is shifted up.

Reply:
Ah, yeah. When looks like when I "squished" it by shrinking the screen for the gif capture it had this weird overflow. I can make it so the short version is literally just "🤗 Model", but this does illustrate that you can very quickly run out of space when you list the full type path inline with the summary viz.

Reply:
<img width="780" alt="Screenshot 2023-06-21 at 3 18 04 PM" src="https://github.com/sematic-ai/sematic/assets/7181589/5ff8a2f0-4415-4cee-9159-6fbaeaacf7cf">

Here's the smaller version ^ . If you're concerned about the chevron wrap-around in general, that is a general problem with the new UI, and not specific to these types.

Reply:
> not specific to these types

apart from the fact that they have longer type paths/names than other examples we've got

Reply:
This can be fixed by adjusting the `flow-shrink` to the path and value UI parts. so each of them will shrink to fit. I can take care of this in a separate PR.  For now, please do whatever to work around the issue.